### PR TITLE
PROD-2140: Describe autofill clusterID in a support case

### PR DESCRIPTION
**Pull request by bobfuru**
@sferich888 PTAL - this PR relates to [OSDOCS-764](https://jira.coreos.com/browse/OSDOCS-764), which is to document changes introduced in [PROD-2140](https://jira.coreos.com/browse/PROD-2140).

I used terminology *Open Support Case* or *Report Bug* in the action to describe both prerelease and GA versions.

### CODE REVIEW

File: modules/support-get-cluster-id.adoc [hunk 1]
```diff
@@ -5,21 +5,25 @@
...
 .Prerequisites
 
 * Access to the cluster as a user with the `cluster-admin` role.
-* The {product-title} CLI (`oc`) installed.
+* Access to the web console or the OpenShift CLI (`oc`) installed.
 
 .Procedure
 
+* To open a support case and have your cluster ID autofilled using the web console:
+.. From the toolbar, navigate to *(?) Help* -> *Open Support Case* or *Report Bug*.
```
Comment (sferich888) [inline modules/support-get-cluster-id.adoc]: You should only see *Open Support Case*. 
You should only see, *Report Bug* if this is OKD.

File: modules/support.adoc [hunk 2]
```diff
@@ -20,7 +20,7 @@ ifdef::openshift-enterprise,openshift-dedicated[]
 When submitting a support case, it is recommended to provide the following information about your cluster to Red Hat Support to aid in troubleshooting:
 
 * Data gathered using the `oc adm must-gather` command
-* The unique cluster ID
+* The unique cluster ID. (This value is autofilled when you navigate to *(?) Help* -> *Open Support Case* or *Report Bug*.)
```
Comment (sferich888) [inline modules/support.adoc]: This 'additonal text' should probably be in a call out or not.

Comment (bobfuru) [reply modules/support.adoc]: I agree, though lines 20-23 are already in an `important note` callout. Also, this note is already conditionalized for Enterprise and Dedicated, so I removed the `Report Bug` ref.

File: modules/support-get-cluster-id.adoc (hunk 1 above)
Comment (bobfuru) [reply modules/support-get-cluster-id.adoc]: Added conditionals

**Comment by bobfuru** [comment]
@sferich888 Can I please get an ack that you are good with changes?

**Comment by bobfuru** [comment]
@yapei PTAL, thanks!

File: modules/support-get-cluster-id.adoc [hunk 3]
```diff
@@ -5,21 +5,30 @@
...
 
 .Prerequisites
 
 * Access to the cluster as a user with the `cluster-admin` role.
-* The {product-title} CLI (`oc`) installed.
+* Access to the web console or the OpenShift CLI (`oc`) installed.
 
 .Procedure
 
+* To open a support case and have your cluster ID autofilled using the web console:
```
Comment (sferich888) [inline modules/support-get-cluster-id.adoc]: Support Case here need to be conditionalized.

Comment (bobfuru) [reply modules/support-get-cluster-id.adoc]: Ah, good point. I updated conditionals so "support case" shows for Enterprise and "bug" shows for Origin.

File: modules/support-get-cluster-id.adoc [hunk 4]
```diff
@@ -5,21 +5,31 @@
...
+* Access to the web console or the OpenShift CLI (`oc`) installed.
 
 .Procedure
+ifdef::openshift-enterprise,openshift-dedicated[]
+* To open a support case and have your cluster ID autofilled using the web console:
+.. From the toolbar, navigate to *(?) Help* -> *Open Support Case*.
+.. The 'Cluster ID' value is autofilled.
+endif::[]
+ifdef::openshift-origin[]
+* To open a bug and have your cluster ID autofilled using the web console:
```
Comment (bobfuru) [inline modules/support-get-cluster-id.adoc]: @sferich888 Should I remove "and have your cluster ID autofilled"? I just tested 4.3 prerelease, and while it opens a BZ, I don't see where it autofills a clusterID.

File: modules/support-get-cluster-id.adoc [hunk 5]
```diff
@@ -5,21 +5,31 @@
...
+.. From the toolbar, navigate to *(?) Help* -> *Report Bug*.
+.. The 'Cluster ID' value is autofilled.
```
Comment (bobfuru) [inline modules/support-get-cluster-id.adoc]: ^^ Same as comment above. Not sure clusterID is autofilled in BZ.

Comment (sferich888) [reply modules/support-get-cluster-id.adoc]: It's put into the bugzilla environment, field. What Bz did you test with?

File: modules/support-get-cluster-id.adoc (hunk 4 above)
Comment (sferich888) [reply modules/support-get-cluster-id.adoc]: No it auto populates a field that's not typically shown. It's an advanced field if I'm not mistakin

Comment (bobfuru) [reply modules/support-get-cluster-id.adoc]: I see it now. It isn't showing in the BZ create bug stage. Once submitted, it does autofill (along with version) in the `Environment` field.

File: modules/support-get-cluster-id.adoc (hunk 5 above)
Comment (bobfuru) [reply modules/support-get-cluster-id.adoc]: Got it. I'm going to update to say that it is autofilled after you click `Submit bug`. Thanks!

**Comment by bobfuru** [comment]
@sferich888 I am using a conditional that shows "Support Case" for `openshift-enterprise` and `openshift-dedicated`. I am using "Report Bug" for `openshift-origin` branch. 

Can you please confirm this is correct in terms of downstream distro support level?

**Comment by sferich888** [comment]
Open support case should also work with online, but I think you have it right.

**Comment by yapei** [comment]
LGTM from QE's point of view

**Comment by bobfuru** [comment]
> Open support case should also work with online, but I think you have it right.

Thanks, Eric. Online doesn't have anything in the "[Support](https://docs.openshift.com/online/starter/support/getting-support.html)" section of docs where it would make sense to include this info, so I'll leave it this way.

**Comment by bobfuru** [comment]
@openshift/team-documentation PTAL. This is for 4.3 and has been approved by SME and QE.

**Comment by ahardin-rh** [comment]
Looks good!

**Pull request merged**